tests/callbacks/test_Manager.cpp

```diff
@@ -6,8 +6,8 @@
 #include "kokkos-utils/impl/type_traits.hpp"
 
 #include "kokkos-utils/callbacks/Events.hpp"
```

Comment:
I think you can get rid of this one too.

Reply:
(The `Events.hpp` include.)

Reply:
OK. Indeed! Done for the other one too.